android/app/src/androidTest/kotlin/coop/polypoly/polypod/FeatureFragmentInstrumentedTest.kt

```diff
@@ -85,16 +86,17 @@ class FeatureFragmentInstrumentedTest {
     @Test
     fun afterStartingAFeature_podObjectEventuallyResolves() {
         val fragmentScenario = launchTestFeature()
-        val polyOut = PolyOutTestDouble(
-            ApplicationProvider.getApplicationContext()
-        )
-        val polyIn = PolyInTestDouble()
-        val podApi: PodApi = PodApiTestDouble(polyOut, polyIn)
+        // wait for the feature container to finish initializing itself
+        Thread.sleep(2000)
```

Comment:
Do you trust it will always need 2 seconds to finish initialization?

Reply:
There's better ways to do this, I guess as a band aid to make the existing tests run again it's fine. I'm currently working on some tests for the notification system that don't need sleeps for this kinda stuff.

Reply:
My comment was not so much in the direction of needing to sleep the thread for a while, but the fact that it needs specifically 2 seconds, and not 1 or three. Tests run in reliably unreliable virtual infra, after all.

Reply:
I agree, what I meant by the _proper_ way to do it is that there are APIs that allow you to wait for certain views to show up (with a long default timeout of like 30s), but my main point is that we need the compilation error and test fixes (even if a bit unreliable) in this PR to further improve on the tests. Revisiting these old tests (also the disabled ones) and figuring out if we still want them and how to improve them is another story.